rnndb/graph/g80_3d.xml

```diff
@@ -67,10 +67,10 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
 			<reg32 offset="0x00" name="ADDRESS_HIGH"/>
 			<reg32 offset="0x04" name="ADDRESS_LOW"/>
 			<reg32 offset="0x08" name="FORMAT" type="G80_SURFACE_FORMAT"/>
-			<reg32 offset="0x0c" name="TILE_MODE">
-				<bitfield low="0" high="3" name="X" />
-				<bitfield low="4" high="7" name="Y" />
-				<bitfield low="8" high="11" name="Z" />
+			<reg32 offset="0x0c" name="BLOCK">
+				<bitfield low="0" high="3" name="WIDTH" type="block_dimensions" />
```

Comment:
umm, do we really need a type for that? If so, you should make a separate type for each of the three coords, since WIDTH has a smaller valid range (0-1) than the others (0-5 IIRC).

I'd much prefer adding min="0" max="5" or something like it.

Better yet, make a bitset type covering all three fields and use that everywhere.